src/auto/accel_group.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (b7f5189) from gir-files (71d73f0)
+// This file was generated by gir (b7f5189) from gir-files (71d73f0+)
```

Comment:
I don't know what that `+` is from as my working directory is clean. Ran submodule update and the submodule is pointed to the correct git version. Manually checked out master and pulled to the latest version and it's still giving me that plus. You ever see that anywhere?